hlxsites/merative2
# feat(microsite): Add Solution Header and Solution Footer blocks used within custom campaign/microsites

## Issue

Fixes https://merative.atlassian.net/browse/MERATIVE-878 & https://merative.atlassian.net/browse/MERATIVE-879

## Description

**New**

- Solution Header `/blocks/solution-header` - This block manages and displays the anchor/table of contents navigation used on a single campaign/microsite.
- Solution Footer `/blocks/solution-footer` - This block manages and displays fragment/related content (See - `website > fragments > footer > footer-small`) (see [link](https://merative.sharepoint.com/:w:/s/Merative.comWebsite/ESfsn8gNRoZJjz2Zx-lV5X4B-TVE1F9z8_rfjCSlCCDrIA?e=jOr4jt)) for the custom footer.

**Changed**

- Fixes grid container of the CTA block


## Design Specs

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=496%3A6561&mode=dev

## Test URLs
 
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/drafts/putra/campaigns/missouri
- After (Changes from this PR): https://feat-solution-header--merative2--proeung.hlx.page/drafts/putra/campaigns/missouri
  
## Testing Instruction

![Screenshot 2023-09-29 at 11 44 38 AM](https://github.com/hlxsites/merative2/assets/1815714/8b023639-7349-4663-9681-f7f878206c69)
![Screenshot 2023-09-29 at 11 44 55 AM](https://github.com/hlxsites/merative2/assets/1815714/214c8b8d-7ff4-4c35-b8a4-024e728bdad1)
![Screenshot 2023-09-29 at 11 44 44 AM](https://github.com/hlxsites/merative2/assets/1815714/a0a4e86d-c916-40fd-a7a5-a3055126161e)

- Visit the "After" deploy preview and ensure that the behaviors and styles of the Header/Nav and Footer align with the design specs.
